src/tests/grpc/clients/ea_restaurant_java_etl_grpc_client_test.py

```diff
@@ -59,3 +63,30 @@ def test_client_refresh_token_with_grpc_client(self):
         self.mocked_java_etl_grpc_client_pb2_grpc.Oauth2ServiceStub().refreshToken.assert_called_with(
             refresh_token_request
         )
+
+    @mock.patch(
+        "src.grpc.clients.ea_restaurant_java_etl_grpc_client.map_mongo_order_status_histories_to_grpc_mongo_order_status_histories"
+    )
+    def test_insert_mongo_order_status_histories_from_python_etl(
+        self, mocked_map_mongo_order_status_to_grpc_mongo_order_status
+    ):
+        mongo_order_status_history_1 = build_order_status_history()
```

Comment:
Why don't you send the id to the builder?

Reply:
Done.